### DISCUSSION TITLE
Design Related Discussions

### DISCUSSION THREAD

**Discussion by orama254** [reactdeveloperske/reactdevske-website#9] [Ideas]
## 💡 Welcome to the Designs Discussion!
  We’re using this Designs Discussions as a place to  share design suggestions and inspirations with the community. We hope that you:
  * Share design ideas.
  * showcase some design inspiration
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with a piece of design that amazed you at some point in your online journey .

**Comment by orama254** [comment]
Wireframes for the first three Pages:

**Landing Page**
---
![reactdevske_Landing](https://user-images.githubusercontent.com/50571688/123117773-b8524580-d44a-11eb-8cd5-7e8b06640986.jpg)
<br>
**About Page**
---
![reactdevske_About](https://user-images.githubusercontent.com/50571688/123117871-cb651580-d44a-11eb-903b-1888932435bc.jpg)
<br>
**Members Page**
---
![reactdevske_Members](https://user-images.githubusercontent.com/50571688/123117932-db7cf500-d44a-11eb-9479-b182ab4652c3.jpg)

**Reply by nassersaazi** [reply]
Can I make a pull request with an implementation for these designs. For example, in relation to issue #15 ?

**Reply by nassersaazi** [reply]
also ,are there any community images that can be used ?

**Reply by orama254** [reply]
Sure you can. Are you good at tests? preferably it would be ideal for some tests written before sending a pull request

You can make a PR with tests and an accompanying implementation

Let me know if you are ok with this

**Reply by nassersaazi** [reply]
alright. not very good with tests but i'll figure it out

**Comment by bmwasonga** [comment]
looks good for a start

**Comment by gmwaniki** [comment]
It looks promising but I think the members section will grow overtime and the page might become too long. Maybe then we can have leaders and top contributors there and a search for individual members

**Reply by orama254** [reply]
I'll consider this

**Comment by stephanieopala** [comment]
Looks great! I also had a similar suggestion for the members page just as @G-nm above has mentioned.

**Comment by Alatharee21** [comment]
For the landing page, there seems to be no CTA, probably to sign up and Login if they've previously signed up.. that could be at the top right corner while the ; "about members events contacts" can be at the middle of that frame.

**Reply by bmwasonga** [reply]
Amazing, 

I look forward to having this conversation with you on a deeper level.

**Reply by DennohKim** [reply]
Kindly clarify on the website main routes; a landing page, about page, members page, events page and  a contacts page.
